util/container/README.md

```diff
@@ -0,0 +1,10 @@
+# HWPE Stream Dockertest Container
+The dockerfile contains the build instructions for the container. It is compatible for cocotb and Verilator. Once built simply go into the `/cocotb` directory and run `pytest`.
```

Comment:
This will not work anymore if you look into my comment about not pulling the repo inside the dockerfile.
I would advise you to mounting the repository as a volume mount in the container instead (this is what snitch also uses!)

Reply:
Will change this hold on :)

Reply:
I took out the `/cocotb` and `pytest` instructions here.